### PR TITLE
Fix admin account creation

### PR DESCRIPTION
**Pull request by dragondaddy**
## Problem

- Impossible to create the admin account
- LDAP prevents from being able to use nomadic identity

## Solution

- Remove LDAP stuff from the code

## PR Status

- [x] Code finished and ready to be reviewed/tested
- [x] The fix/enhancement were manually tested (if applicable)

## Automatic tests

Automatic tests can be triggered on https://ci-apps-dev.yunohost.org/ *after creating the PR*, by commenting "!testme", "!gogogadgetoci" or "By the power of systemd, I invoke The Great App CI to test this Pull Request!". (N.B. : for this to work you need to be a member of the Yunohost-Apps organization)

### CODE REVIEW

**Comment by dragondaddy** [comment]
!testme

**Comment by dragondaddy** [comment]
!testme

**Comment by pepecyb** [comment]
This is an important bug fix because it is no longer possible to install the app in a usable way without it. As the tests are running smoothly, it should be adopted as soon as possible.

**Comment by morphtown** [comment]
I can confirm that it is not possible to get the verification token for the initial account via mail. It just won't be sent.

File: scripts/_common.sh [hunk 1]
```diff
@@ -5,7 +5,7 @@
 #=================================================
 # PHP APP SPECIFIC
 #=================================================
-YNH_PHP_VERSION="8.2"
+YNH_PHP_VERSION="8.3"
```
Comment (ericgaspar) [inline scripts/_common.sh:8]: this is not needed. the PHP version is taken from `[recources.apt]`

**Comment by ericgaspar** [comment]
is this tested? I don't have it installed in my server.

**Comment by dragondaddy** [comment]
I'll test it in a matter of minutes, just need to put the kids to sleep first. I'll keep you updated.

**Comment by dragondaddy** [comment]
Ok it's done. Fresh install on my server worked fine, the hub is usable. Also tested with package checker (on my computer) all tests are OK.

**Comment by dragondaddy** [comment]
!testme

**Comment by ericgaspar** [comment]
and what about updating from previous version? will user be able to connect?

**Comment by dragondaddy** [comment]
That should not be an issue. But I'll give it a try later today, just to be sure.

**Comment by dragondaddy** [comment]
Okay so after installing the previous package version and creating a few accounts & posts, I can confirm that the login still works after the update.

**Comment by dragondaddy** [comment]
!testme

**Pull request merged**